### ISSUE TITLE
Store new receipt.topicRunningHashVersion

### ISSUE THREAD

**Issue by steven-sheehy** [hashgraph/hedera-mirror-node#688]
**Problem**
HAPI will soon calculate a topic's running hash differently. To support the old format, a new `receipt.topicRunningHashVersion` has been added. We should store this new field in the database in case it is needed by clients.

**Solution**
- Add a new field to `topicmessage` named `runningHashVersion` of type `smallint` (to save space)
- Enhance code to populate new field

**Alternatives**

**Additional Context**
Split this out and depends on initial R5 protobuf changes to be made by #722

**Comment by steven-sheehy** [comment]
Blocked on protobufs being updated

**Comment by Nana-EC** [comment]
- Add migration version to add runningHashVersion column to topic_message. 
- Update parser to extract runningHashVersion from record.
- Update parser to write extracted runningHashVersion to topic_message table

Q's
- Is proto already updated on service side?
- What's the default runningHashVersion or starting version for existing data?

**Comment by apeksharma** [comment]
> For migration we can set old data to 0

It's correct only for those migrating before R5 release (us), and would be wrong if someone migrates after the release.

**Comment by Nana-EC** [comment]
> It's correct only for those migrating before R5 release (us), and would be wrong if someone migrates after the release.

Agreed, details would be in the migration implementation. Set 0 only on null/empty value.
